src/app/dim-ui/PressTip.m.scss

```diff
@@ -141,18 +128,28 @@ $horizontal-padding: 11px;
       padding: 3px 6px;
     }
 
-    .arrow {
-      --length: #{$minimal-arrow-size * 0.5};
```

Comment:
It looks like the minimal tooltips aren't getting their reduced arrow size anymore:

![dim-minimal-tooltip-arrow-size](https://github.com/DestinyItemManager/DIM/assets/17512262/3508c8f3-8bf0-4853-a925-19f142c256c9)

Reply:
This should be fixed now. 

I centralised the arrow size variables in `_variables.scss`, and added a local variable in `PressTip.m.scss` that inherits the size depending on whether it's a normal vs minimal tooltip. 

![image](https://github.com/DestinyItemManager/DIM/assets/1396158/cacef991-79d8-4fa3-b944-53d709c536ab)

A side-effect of the tooltip refactoring squeezed the spacing making the tip of the arrow sit on (instead of adjacent to) the target. I've remedied this by introducing a margin the size of the arrow on the tooltip container — however i'm not sure if there might be a cleaner/better way to do this — do you have any thoughts?

I'm also not sure how the top/bottom variants of the minimal tooltip render as i've been unable to invoke them in DIM. Any idea where I can see examples of these?

Reply:
> I'm also not sure how the top/bottom variants of the minimal tooltip render as i've been unable to invoke them in DIM. Any idea where I can see examples of these?

I can't get them to naturally show up either, but you can override the placement [here](https://github.com/guise/DIM/blob/sass-to-css-variables/src/app/dim-ui/PressTip.tsx#L101).

```diff
-    placement,
+    placement: 'top',
```

It looks like the horizontal alignment is incorrect for both `top` and `bottom`, and the vertical alignment is incorrect for `top`.

![dim-minimal-tooltip-top-bottom-alignment](https://github.com/DestinyItemManager/DIM/assets/17512262/0b07c32c-b34e-4cfe-94e0-bcecebeffdb6)

Reply:
Thanks. Looks like this can also be checked by modifying in-page with devtools

Reply:
The arrow positioning for minimal tooltips is still a little bit off:

![dim-minimal-tooltip-top-bottom-alignment-2](https://github.com/DestinyItemManager/DIM/assets/17512262/3ea07560-7be5-4ed6-8f7a-cb9dc758815c)

Did a little bit of digging and it's due to Popper calculating the wrong translation to apply to the arrow, because we're [zeroing out the `width` and `height`](https://github.com/guise/DIM/blob/sass-to-css-variables/src/app/dim-ui/PressTip.m.scss#L91-L92). Rather than try to fight with Popper's automatic arrow positioning, I suggest we let the `width` and `height` be automatically calculated and let Popper calculate the correct offset.

I've pushed a possible solution [here](https://github.com/DestinyItemManager/DIM/commit/889af6bb46bcbae6919629cdffb15fa1f760434f) which seems to be working ok.

Reply:
Awesome, thanks. 

Turns out my local previewing of styles by adjusting the tooltip placement in devtools is not reliable since it doesn't apply the inline styles injected by the js correctly. 

I've merged in your solution 👍🏽